entries/b/breww.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Breww": {
+    "domain": "breww.com",
+    "tfa": [
+      "totp",
+      "webauthn"
+    ],
+    "documentation": "https://docs.breww.com/setting-up-2-factor-authentication-2fa",
```

Comment:
The documentation element cannot link to pages that requires authentication to view.
If no publicly accessible documentation is available, please attach screenshots of the 2FA setup and/or login process.
> [!NOTE]
> Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment like this:
![screenshot](https://user-images.githubusercontent.com/3535780/187552643-08eea7e8-e6c6-4873-b31c-0faf4347f69f.gif)